### PR TITLE
aws_securityhub list_members date format fixes

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [#48233](https://github.com/demisto/etc/issues/48233)

## Description
The response for **aws-securityhub-list-members** commands is retrieved with dates in a date-time format which the CSP tries to convert to string using JSON-conversion.
Therefore, I added a helper function to convert the date-time objects to str using the **strftime** command.
I reproduced the problem and succeeded in executing the command using my fixes.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

### CODE REVIEW

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py [hunk 1]
```diff
@@ -610,12 +610,22 @@ def list_members_command(client, args):
...
     human_readable = tableToMarkdown(table_header, response.get('Members', []))
     return human_readable, outputs, response
 
+def convert_members_date_type(members):
+    new_ls = []
+    for member in members:
+        if isinstance(updated_at:=member['UpdatedAt'], datetime):
+            member['UpdatedAt'] = updated_at.strftime('%Y-%m-%d %H:%M:%S')
+        if isinstance(invited_at:=member['InvitedAt'], datetime):
+            member['InvitedAt'] = invited_at.strftime('%Y-%m-%d %H:%M:%S')
```
Comment (ShahafBenYakir) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py:626]:
```suggestion
        if isinstance(updated_at:=member.get('UpdatedAt'), datetime):
            member['UpdatedAt'] = updated_at.isoformat()
        if isinstance(invited_at:=member['InvitedAt'], datetime):
            member['InvitedAt'] = invited_at.isoformat()
```

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py [hunk 2]
```diff
@@ -610,12 +610,23 @@ def list_members_command(client, args):
...
+    table_header = 'AWS SecurityHub ListMembers' 
     human_readable = tableToMarkdown(table_header, response.get('Members', []))
     return human_readable, outputs, response
 
+def convert_members_date_type(members):
+    new_ls = []
+    for member in members:
+        if isinstance(updated_at:=member.get('UpdatedAt'), datetime):
+            member['UpdatedAt'] = updated_at.isoformat()
+        if isinstance(invited_at:=member['InvitedAt'], datetime):
```
Comment (ShahafBenYakir) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py:625]:
```suggestion
        if isinstance(invited_at:=member.get('InvitedAt'), datetime):
```

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py (hunk 1 above)
Comment (YuvHayun) [reply Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py]: I think using **strftime** is better
**strdtime** yield this format: 2018-12-09 13:59:10
while **isoformat** yield this format: 2018-12-09T13:58:06.723000+00:00

Comment (ShahafBenYakir) [reply Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.py]: ISO format is the standard when it comes to time formats, it is the default format in the datetime library and it is also the default format that demisto is using.

File: Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py [hunk 3]
```diff
@@ -166,3 +167,24 @@ def test_fetch_incidents(mocker):
...
+        - mock aws_session
+    When:
+        - Running list_members_command
+    Then:
+        - Ensure that the command was executed correctly. In particular, ensure that the datetime fields are convereted to str.
+    """
+    aws_client = AWSClient("reg", "", "", 900, "p", "mock_aws_access_key_id", "mock_aws_secret_access_key", "", "", 3)
+    client = aws_client.aws_session(service='securityhub', region="reg", role_arn='roleArnroleArnroleArn',
+                                    role_session_name='roleSessionName')
+    now_time = datetime.datetime.now()
```
Comment (ShahafBenYakir) [inline Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub_test.py:184]: For consistency between rubs, let's give here a constant time (not now). Something like this:
```suggestion
    time_val = dateparser.parse('2022-01-01',settings={'TIMEZONE': 'UTC'})
```

File: Packs/AWS-SecurityHub/ReleaseNotes/1_1_16.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+- **AWS - Security Hub**
+- Updated the Docker image to: *demisto/boto3py3:1.0.0.29538*.
+- Fixed an issue where **list_members_command** failed due to response containing values of type datetime.
```
Comment (ShahafBenYakir) [inline Packs/AWS-SecurityHub/ReleaseNotes/1_1_16.md:5]: Get yourself familiarized with the documentation styling guidelines https://xsoar.pan.dev/docs/documentation/release-notes#entity-styling
```suggestion
- Fixed an issue where the ***aws-securityhub-list-members*** failed to parse API responses that contained datetime objects.
- Updated the Docker image to: *demisto/boto3py3:1.0.0.29538*.

```

**Review by ShahafBenYakir — APPROVED** [review]
Good. Small fixes

**Pull request merged**